cfgov/teachers_digital_platform/documents.py

```diff
@@ -0,0 +1,136 @@
+from django_elasticsearch_dsl import Document, fields
+from django_elasticsearch_dsl.registries import registry
+
+from search.elasticsearch_helpers import strip_html
+from teachers_digital_platform.models.pages import ActivityPage
+
+
+mtm_fields = [
+    "building_block",
+    "school_subject",
+    "topic",
+    "grade_level",
+    "age_range",
+    "student_characteristics",
+    "activity_type",
+    "teaching_strategy",
+    "blooms_taxonomy_level",
+    "jump_start_coalition",
+    "council_for_economic_education",
+]
+
+
+@registry.register_document
+class ActivityPageDocument(Document):
+    """Index an ActivityPage's fields and relations."""
+
+    text = fields.TextField(attr='summary', boost=10)
+    title = fields.TextField(attr='title', boost=10)
+    date = fields.DateField(attr='date')
+    big_idea = fields.TextField()
+    essential_questions = fields.TextField()
+    objectives = fields.TextField()
+    what_students_will_do = fields.TextField()
+    related_text = fields.TextField()
+    # Foreign-key field:
+    activity_duration = fields.KeywordField()
+    # MTM fields
+    activity_type = fields.KeywordField()
+    age_range = fields.KeywordField()
+    blooms_taxonomy_level = fields.KeywordField()
+    building_block = fields.KeywordField()
+    council_for_economic_education = fields.KeywordField()
+    grade_level = fields.KeywordField()
+    jump_start_coalition = fields.KeywordField()
+    school_subject = fields.KeywordField()
+    student_characteristics = fields.KeywordField()
+    teaching_strategy = fields.KeywordField()
+    topic = fields.KeywordField()
+
+    class Index:
+        name = 'teachers-digital-platform'
```

Comment:
You will need to incorporate the [new helper function](https://github.com/cfpb/consumerfinance.gov/blob/main/cfgov/search/elasticsearch_helpers.py#L56) to prefix the environment to this index name.

Reply:
> You will need to incorporate the [new helper function](https://github.com/cfpb/consumerfinance.gov/blob/main/cfgov/search/elasticsearch_helpers.py#L56) to prefix the environment to this index name.

Thanks! [Fixed](873f8d15fea247d78508c39b8c9e28da60d1894b)